### DISCUSSION TITLE
Debug symbols for prebuilt strongswan binaries

### DISCUSSION THREAD

**Discussion by sharathbhatp** [strongswan/strongswan#2455] [Q&A]
Hi, 

Is there a way we can get debug symbols for prebuilt strongswan binaries using apt? We use strongswan 5.9.5 on ubuntu22.

**Comment by tobiasbrunner** [comment]
Sure, please refer to the [official Ubuntu documentation](https://ubuntu.com/server/docs/debug-symbol-packages).

**Reply by sharathbhatp** [reply]
Thanks. According to that documentation, we can get dbg symbols from PACKAGE-dbgsym. Any idea why `strongswan-dbgsym` cannot found by apt?

**Reply by tobiasbrunner** [reply]
Because that's a metapackage that only depends on other packages, it does not contain any files.

**Comment by sharathbhatp** [comment]
This helped https://ubuntu.com/server/docs/about-debuginfod
